data/filters/url-param-remover.yaml

```diff
@@ -0,0 +1,26 @@
+title: Remove unneeded URL parameters
+params:
+  - name: url-params
+    description: Parameters to hide
+    type: list
+    default: [ "utm_source", "utm_medium", "utm_campaign", "utm_content" ]
```

Comment:
These values are already covered by the [AdGuard URL Tracking Protection](https://github.com/AdguardTeam/AdguardFilters/tree/master/TrackParamFilter/sections) list, which is shipped by uBO (although disabled by default). I think we should acknowledge this in the filter description and encourage people to use that list, as they'll benefit from community updates.

They still make good example values, as they are pretty innocuous if enabled by anyone.